src/main/java/com/softserve/academy/spaced/repetition/service/cacheScheduler/CacheScheduler.java

```diff
@@ -0,0 +1,26 @@
+package com.softserve.academy.spaced.repetition.service.cacheScheduler;
+
+import com.softserve.academy.spaced.repetition.service.CategoryService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CacheScheduler implements CommandLineRunner {
+
+    private static final Logger logger = LoggerFactory.getLogger(CacheScheduler.class);
+
+    @Autowired
+    private CategoryService categoryService;
+
+    @Override
+    public void run(String... args) throws Exception {
+        logger.info("Started adding categories to cache...");
+        categoryService.getTopCategory();
```

Comment:
you add categories to cashe, but what about decks

Reply:
decks cache will be much bigger